package.json

```diff
@@ -15,10 +15,12 @@
     "start": "cross-env NODE_OPTIONS=\"--no-warnings --loader=ts-node/esm\" node examples/ding-dong-bot.ts",
     "start:js": "node examples/ding-dong-bot.js",
     "start:walnut": "cross-env WECHATY_LOG=verbose WECHATY_PUPPET=wechaty-puppet-walnut npm start",
-    "start:service": "cross-env WECHATY_LOG=verbose WECHATY_PUPPET=wechaty-puppet-service npm start",
+    "start:service": "cross-env WECHATY_LOG=verbose WECHATY_PUPPET=wechaty-puppet-service WECHATY_PUPPET_SERVICE_NO_TLS_INSECURE_CLIENT=true npm start",
```

Comment:
`WECHATY_PUPPET_SERVICE_NO_TLS_INSECURE_CLIENT` should not be set in official examples.

Instead, the Puppet Service should start supporting the TLS to maximum the security.